### PR TITLE
feat(I18n): add zh-TW.yml

### PR DESCRIPTION
**Pull request by run26kimo**
add I18n zh-TW yml

### CODE REVIEW

File: config/locales/zh-TW.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  logo-title: Vuash Mark (Paper Airplane)
+
+  # Site Footer
+
+  footer:
+    features:
+
+      safety:
+        title: Safety - 安全
+        text: Your message is encrypted right away with a secret key that isn't stored in our database.
```
Comment (rmasoni) [inline config/locales/zh-TW.yml]: Hey, thanks a lot for the contribution!

I have a question, though: why are these parts still in english?

Comment (run26kimo) [inline config/locales/zh-TW.yml]: i want keep there english.
btw 
vuash is a nice idea.

**Comment by rmasoni** [comment]
@run26kimo Can you please review these changes: https://github.com/current/vuash/commit/5901e130c43f2417b0a623f35ca2e6cc457bda1d

Does it still make sense in chinese?

**Comment by rmasoni** [comment]
I just checked with a chinese friend and she said it's alright. Maybe a bit too colloquial, but it might do for now.

I'll merge the new branch into master and close this PR.

Thanks again for the contribution!

**Pull request closed without merging**